packages/cadl-ranch-specs/http/resiliency/srv-driven-2/main.cadl

```diff
@@ -104,11 +104,11 @@ Show that you can call a POST HTTP endpoint.
 This test now accept both image/jpeg and applicat/json and is expected keeping backward compat with srv-driven-1.
 Pass the JSON: `{"url": "http://example.org/myimage.jpeg"}` or a binary with content-type image/jpeg. The server do not check the binary.
 """)
-  @route("/parameters/{contenttype}")
+  @route("/parameters/{contentTypePath}")
   @doc("POST a JSON or a JPEG")
   @post
   postParameters(
-    @path contenttype: "json" | "jpeg",
+    @path contentTypePath: "json" | "jpeg",
```

Comment:
curious why rename `contenttype` to `contentTypePath` ?

Reply:
I renamed it because this conflicts with the standard `Content-Type` header parameter we generate whenever we are sending a body to the service